articles/synapse-analytics/spark/spark-dotnet.md

```diff
@@ -23,18 +23,42 @@ You can analyze data with .NET for Apache Spark through Spark batch job definiti
 
 Visit the tutorial to learn how to use Azure Synapse Analytics to [create Apache Spark job definitions for Synapse Spark pools](apache-spark-job-definitions.md). If you haven't packaged your app to submit to Azure Synapse, complete the following steps.
 
-1. Run the following commands to publish your app. Be sure to replace *mySparkApp* with the path to your app.
+1. Configure your dotnet application dependencies for compatibility with Synapse Spark.
+The required .NET Spark version will be noted in the Studio interface under your Spark Pool configuration. 
+![image](https://user-images.githubusercontent.com/12169942/161816341-ca13b48f-fb18-49bd-a61b-6af60bf9846e.png)
```

Comment:
Images need to be hosted in docs, not on uploaded Github content.

Reply:
Image files go here: https://github.com/MicrosoftDocs/azure-docs/tree/main/articles/synapse-analytics/spark/media/<folder for this page>